### DISCUSSION TITLE
Metadata & date options

### DISCUSSION THREAD

**Discussion by MaxPancake** [tbabej/taskwiki#355] [Q&A]
Long-time user of vimwiki & taskwarrior, and recently got taskwiki to work, but having trouble w/ metadata for tasks. According to doc, the '--' divider should do the trick, as in:
`* [ ] This has project home -- project:Home`
But it doesn't work for me and I can't find out why. I got the dev branch of vimwiki, taskwarrior 2.5.3, tasklib 2.3.0.

Another question: is there a more convenient way to enter the due date like tomorrow, +5days, the sorts of things allowed in taskwarrior?

**Comment by liskin** [comment]
` -- …` only works for new tasks. Existing tasks can be modified via <kbd>\<Leader\>tm</kbd>.

> is there a more convenient way to enter the due date …

There's omnicompletion (<kbd>C-X O</kbd>) for metadata after ` -- ` and completion (<kbd>Tab</kbd>) when modifying tasks. Is that the kind of convenience you had in mind?

**Reply by MaxPancake** [reply]
I swear the "--" thing didn't work until just now 🤣  ok all the things you mentioned work now except the `C-X O` part.. is it `Ctrl`+`X`+`O`? I tried that and vim just took me through a bunch of files I recently edited.. ??

**Reply by liskin** [reply]
It's <kbd>C-X</kbd>, then <kbd>O</kbd>, in insert mode. Check that `:echo &omnifunc` shows `taskwiki#CompleteOmni` and that `:imap <C-X>` doesn't list a mapping overriding vim's default.

**Reply by MaxPancake** [reply]
ok will check that out. thanks!
